src/styles/utils/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
```

Comment:
you can keep media in BEM block file (`catalog`)

Reply:
done